### PR TITLE
[MM-49751] - Turn off Inactive Server Email

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR removes all the code changes that were responsible for sending an email for server inactivity

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-49751

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by ConorMacpherson — APPROVED** [review]
LGTM, just needs translations fixed.

File: i18n/en.json [hunk 1]
```diff
@@ -7501,23 +7453,23 @@
...
   {
     "id": "ent.elasticsearch.create_client.client_cert_missing",
-    "translation": "Could not open the client certificate file for Elasticsearch"
+    "translation": ""
   },
   {
     "id": "ent.elasticsearch.create_client.client_key_missing",
-    "translation": "Could not open the client key file for Elasticsearch"
+    "translation": ""
   },
```
Comment (nickmisasi) [inline i18n/en.json:7473]: Are these intentional?

**Review by nickmisasi — APPROVED** [review]
no blocking question

**Pull request closed without merging**